### PR TITLE
Ep_power axis based on phalf

### PR DESCRIPTION
**Pull request by guinneak198**

### CODE REVIEW

File: examples/combined_ODNP.py [hunk 1]
```diff
@@ -62,8 +62,12 @@
...
 # }}}
 # {{{Power settings
-dB_settings = gen_powerlist(
-    config_dict["max_power"], config_dict["power_steps"] + 1, three_down=True
+dB_settings = Ep_spacing_from_phalf(
+    est_phalf = config_dict['guessed_phalf'],
+    max_power = config_dict["max_power"], 
+    config_dict["power_steps"] + 1, 
+    min_dBm_step = config_dict['min_dBm_step'],
+    three_down=True
```
Comment (jmfranck) [inline examples/combined_ODNP.py:70]: I think that this is good, but I ended up looking at the function for Ep_spacing_from_phalf --> can you give a link to where that math is on slack?  i.e. how do we justify the math there?

Comment (jmfranck) [reply examples/combined_ODNP.py:70]: I'm saying that I want that info so that I can add the math to the docstring (or else this will never get done)

Comment (guinneak198) [reply examples/combined_ODNP.py:70]: https://jmfrancklab.slack.com/archives/CLMMYDD98/p1668103766923999

Comment (jmfranck) [reply examples/combined_ODNP.py:70]: https://jmfrancklab.slack.com/archives/CLMMYDD98/p1680712474001329?thread_ts=1668103766.923999&cid=CLMMYDD98

File: SpinCore_pp/power_helper.py [hunk 2]
```diff
@@ -18,33 +18,55 @@ def Ep_spacing_from_phalf(
...
+    calculate
+    :math:`dl = \sqrt{dp^2 + df^2} = dp\sqrt{\left(\frac{dp}{dp}\right)^2+\left(\frac{df}{dp}\right)}=dp\sqrt{1+\frac{df}{dp}}`
+    and then integrate :math:`l(p) = \int_{p=0}^{p=p_{max}} dl`,
+    which gives the length along the curve (as though we placed a string along the curve).
+
+    Note that we very rarely plot so that our power axis and enhancement
+    axis are plotted with an equal aspect ratio,
+    so we use the max power, max enhancement, and aspect ratio to
+    determine the spacing of powers that fits the above description
+    *as plotted*.
```
Comment (jmfranck) [inline SpinCore_pp/power_helper.py:47]: see that this makes sense to you, and request any changes!

Comment (guinneak198) [reply SpinCore_pp/power_helper.py]: edited with small typo fix but this makes sense (good note on the string with evenly marked lengths!) but looks good!

**Pull request merged**